### DISCUSSION TITLE
Tensile test specimen (LS-DYNA) format simulation result issue

### DISCUSSION THREAD

**Discussion by CAE4U** [OpenRadioss/OpenRadioss#2375] [Contributors]
Hello Users/Contributors,
We are open source simulation enthusiasts. 
We have downloaded and installed OpenRadioss Linux version. The tool installed nicely along with python gui. As a sample case, we ran the tensile test specimen model (LS-DYNA format), downloaded from example models. The simulation completed successfully,but the result file shown an abnormal stretch of a node.
The installation and simulation can be viewed in the below link.
[https://youtu.be/YRwlLnXADYQ](url)
we would appreciate your comments to identify the source of the issue.

Cheers,
CAE4U

**Comment by MarianBulla** [comment]
Hi CAE4U,
thanks for you interest on OpenRadioss, also for the very nice YouTube video :-)
Regarding the results, there are 2 versions of this tension test:
- One without runture (which you have)
- one with rupture criterion (see below).

You can also easily add the failure limit defined by max. plastic strain in your model, by adjusting this limit in the material card ... just change the value from zero (= no failure) to 0.5 (= element will be switched off, when reaching 50% plastic strain).

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/052b0eee-2741-4947-b35d-a421b2604f28)

[zug_test3_RS.zip](https://github.com/OpenRadioss/OpenRadioss/files/14406604/zug_test3_RS.zip)

In PARAVIEW, you have then to add the filter, to switch-off out the eroded elements.

Hope it helps you ?
Thanks and best regards,
Marian

**Comment by CAE4U** [comment]
Hi Marian,
Thank you for your swift response. The modification do reflect the eroded elements, how ever the issue seems to persist.
Please refer to below picture.we could still see, stretched elements, we are not sure if this is an artifact. Clarification on the same could be very helpful.
![Screenshot from 2024-02-26 22-10-16](https://github.com/OpenRadioss/OpenRadioss/assets/161336918/f562aa9f-7ed3-4475-81dc-e8cceb7e6f2e)

Best Regards,
CAE4U

**Comment by MarianBulla** [comment]
Hi CAE4U,

this is simple to be answered :-) 
It is a "virtual element" created by the solver to measure the section-force:

It depends on your post-prozessor, how it will be displayed.
HyperView is switching it OFF, by default.



![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/3449efad-a989-4e2f-aba2-04246352d3fc)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/5738d61d-13cb-4227-af75-ca812a5dd262)


So, you can ignore it, if you can't switch it off, in ParaView. It only initializes the elements, where the force is measured, for beeing able to plot the stress-vs-strain curve from this virtual tension-test.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by CAE4U** [comment]
Hi Marian,
Yes of-course, adding one more filters in Paraview did the trick. This is bit counter intuitive, we are getting used to it :-).
Hyperview can be a good option, when we checked we found, the supported platform is Linux x86_64 (64-bit) RHEL/CentOS 8.2 SLES 15 SP1. Are you aware of any workaround to use it in Ubuntu?

Regards,
CAE4U

**Comment by MarianBulla** [comment]
Hi CAE4U,

in this case (the simple tension-test example) you even do not need the section-cut-force, because the force is also requested and measured at both rigid-bodies (at both ends, where the specimen is fixed and on the other end, where it is pulled).

So you can use this resultant force from this virtual tests (dividing by the initial section area will give you the engineering stress):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/8c64c44d-db02-4770-9515-c1d1faa47fb9)

For you model, you can simple delete this keyword or comment it out, here:

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/30b4646c-0f1a-4206-803b-f5608abf66d5)

For more complex structures (like the bumper-beam example) it is a different story.

Regarding Ubuntu.....it should work. I'm not aware of any issues with HyperView running under UBUNTU, but I will double-check. 

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by CAE4U** [comment]
Hi Marian,
Thank you for your suggestions. Yes, they are good details to know.

We are actually trying to explore the features supported by the software with this simple example, so that we can smoothly scale up. We feel knowing the tweaks, workarounds and solutions from simple models are easy to relate to the solutions, and can be very handy while troubleshooting  complex models. We hope using the software to simulate more complex models, soon!!.

Regards,
CAE4U.

**Comment by MarianBulla** [comment]
Hi CAE4U,
regarding HyperView on Ubuntu, it depends on the supported graphics driver ... It is out-of-scope for OpenRadioss, sinze it touches the GUI.
In case of any issues, you can reach out to the official Altair support. 

Hope it helps you ?

Thanks and best 
Marian

**Comment by CAE4U** [comment]
Hi Marian, 
Thank you for your response.
we feel it would be good idea to stick with Paraview for the moment. it would avoid too many unknowns to begin with :-).

Best Regards,
CAE4U

**Reply by MarianBulla** [reply]
Hi CAE4U, 
yes, I fully agree with you!

Start simple....not overloading new users with too much complexity.

Feel free to reach out to me, in case of any further questions related to OpenRadioss.

And HAVE FUN with this solver :-) 

Thanks and best regards,
Marian

**Reply by CAE4U** [reply]
Hi Marian,
Glad to know you are in agreement with our approach. Along with FUN, we hope it would make more SENSE with the solver.

Regards,
CAE4U.
